### ISSUE TITLE
"lib" is not a known member of module (reportGeneralTypeIssues)

### ISSUE THREAD

**Issue by danielgordon10** [microsoft/pyright#4649]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of the problem.

**Describe the solution you'd like**

**Additional context**

I get this when I do something like 
```python
import numpy as np
np.lib.pad(...)  #  error: "lib" is not a known member of module (reportGeneralTypeIssues)
```

The annoying thing is I know it's a false positive pretty much every time I see this, but they all fall under "reportGeneralTypeIssues" which is a big umbrella. I'd like to suppress all of this kind of warning without suppressing other type issues. Is there a way to do that?

Side-note: I am using numpy 1.20.2 which has typestubs but for some reason pretty much every subpackage has this issue (numpy.random, numpy.linalg, numpy.testing etc).

**Comment by erictraut** [comment]
Yes, this is a bug in that version of numpy. Please file bugs against numpy for these issues so they get fixed.

You can use a `# type: ignore` in specific cases where this occurs.

**Comment by danielgordon10** [comment]
I know I can ignore a single line but I would like to be able to suppress all warnings of this type. I'd like to add a line like 
`"reportUnknownModuleMember": false,`
in my pyrightconfig.json.
I realize this is not a bug with pyright, but all the same here would be the easier place to fix it since it could be from a variety of packages that have typing issues.

**Comment by erictraut** [comment]
Pyright already supports about 60 different diagnostic rules. Each needs to be maintained and documented. In the past, we have split out sets of diagnostics into their own diagnostic rule. This particular one is very specific, so I'm not sure it makes sense to have a rule of its own. I understand how this would help you in your particular case, but I don't know if this is generally applicable. For now, I'm not going to split it out. It's something we might consider in the future if we receive additional feedback from other pyright users.

**Comment by danielgordon10** [comment]
Is there a way to "trick" pyright into ignoring it other than `# type: ignore` I know normally I could do `import a.b` at the top of the file and it would work, but numpy is complicated since it's very standard to do `import numpy as np` and there is no way I know of to then do `import np.lib`

**Comment by erictraut** [comment]
You can create your own local numpy type stub that simply consists of the following:

```python
def __getattr__(name: str) -> Any: ...
```

This will eliminate all such diagnostics for the module, but it will also eliminate all type checking for symbols imported from the module.

Incidentally, newer versions of numpy fix the problem with `lib`. So you may be able to simply upgrade to a newer version and resolve many of the problems you're hitting.

**Comment by danielgordon10** [comment]
Thanks, I will try that, though I know there are other libraries with similar issues. Hopefully they aren't as commonly used as numpy and we can just type: ignore them.
